rfcs/0000-unified-configuration.ftd

```diff
@@ -0,0 +1,188 @@
+-- import: fastn.com/rfcs/lib
+-- import: fastn-community.github.io/bling/note
+
+-- lib.rfc: RFC: Unified Configuration
+id: unified-config
+status: proposal
+
+TODO: Write a brief summary of the RFC here.
+
+
+-- lib.motivation:
+
+;; TODO: add links to the following paragraph
+Currently, `fastn` supports configuration through environment variables, and in
```

Comment:
Lets mention cli arguments also (eg port, external-js etc).

Reply:
[resolved in this commit](https://github.com/fastn-stack/fastn.com/pull/523/commits/5fcd2fd435f4479915c450606a4c78f19e9e4a61)